x-pack/plugins/security/server/authorization/api_authorization.ts

```diff
@@ -48,7 +48,7 @@ export function initAPIAuthorization(
 
     if (security) {
       if (isAuthzDisabled(security.authz)) {
-        logger.warn(
+        logger.debug(
```

Comment:
__question:__ Is there any value in this log entry at all? I mean, we already have HTTP logs if we want to know which endpoint is being called, and if we want to check whether this endpoint has authorization enabled or disabled, we just need to know the Kibana version, since the definition is mostly static and set in the code.

If, for some reason, we want to keep this as a debug log, it would make sense to remove the `request.url.search` part, as it’s not relevant to the authorization decision (it's essentially based on `path`) and might _potentially_ contain sensitive data that we don’t want to record in logs.

Reply:
tbh I think we are okay to delete it, don't see any issues with that, we are just generating additional noise with it

Reply:
removed in 4493b87f09331a99d17d20a8dab67fe0bc9cb974